boards/efm32g-st700/include/board.h

```diff
@@ -0,0 +1,65 @@
+
+
```

Comment:
Should this be the same as in all the other files?
ie.
/*
 * Copyright (C) 2014 Freie Universität Berlin
 *
 * This file is subject to the terms and conditions of the GNU Lesser General
 * Public License v2.1. See the file LICENSE in the top level directory for more
 * details.
 */